### PR TITLE
ARC-1432 Standardise ESLint rules across the app

### PR DESCRIPTION
**Pull request by yvonnevictoria**
Updates in this PR:
- Standardise ESLint rules across app
- Add @forge/bridge to project
- Update all spaces to tabs
- Remove dangling commas

### CODE REVIEW

File: app/.eslintrc.json [hunk 1]
```diff
@@ -1,37 +1,83 @@
...
-    "implicit-arrow-linebreak": 0,
-	"arrow-body-style": 0,
-	"max-len": [
-	  "error",
-	  {
-		  "code": 120,
-		  "ignoreStrings": true
-	  }
+	"extends": [
+		"airbnb-base",
```
Comment (atrigueiro) [inline app/.eslintrc.json:3]: If anyone was curious about what this is: https://www.npmjs.com/package/eslint-config-airbnb-base

File: app/src/common/error.ts [hunk 2]
```diff
@@ -1,30 +1,30 @@
 /* eslint-disable max-classes-per-file */
 export class JenkinsAppError extends Error {
-  constructor(message: string) {
-    super(message);
-  }
+	constructor(message: string) {
```
Comment (atrigueiro) [inline app/src/common/error.ts:3]: What was the decision we came up here again? 8 spaces? 2 tabs?

**Comment by atrigueiro** [comment]
nit: We should have a Jira task for formating these, mentioning which changes we're making to our lint rules. The key you're linking here has nothing to do with standardising linting rules.

**Comment by atrigueiro** [comment]
- [ ] Tested in stagin

File: app/src/common/error.ts (hunk 2 above)
Comment (yvonnevictoria) [reply app/src/common/error.ts:3]: Tabs and the `.editorconfig` will let you set it to however many spaces you want those tabs to be :) GitHub sets the tabs to be 4 spaces wide

https://hello.atlassian.net/wiki/spaces/PF/pages/1424951560/Arc+Coding+Conventions

File: app/src/webtrigger/handle-jenkins-request.test.ts [hunk 3]
```diff
@@ -2,11 +2,15 @@
  * This is an integration test of the whole code path from the webtrigger through the app
  * to the Jira API.
  */
+
+/* Updating error to warning as there are reasonable exceptions to the rule in this file */
```
Comment (yvonnevictoria) [inline app/src/webtrigger/handle-jenkins-request.test.ts:6]: https://softwareteams.atlassian.net/browse/ARC-1433
Added ticket to address this rule - move JWT tokens to separate file and disable lint errors there instead

**Review by joshkay10 — APPROVED** [review]

**Review by krazziekay — APPROVED** [review]

**Pull request merged**